server/controllers/events/testfixtures/test-repos/repo-config-file/exp-output-autoplan.txt

```diff
@@ -4,10 +4,7 @@ Ran Plan for 2 projects:
 1. dir: `infrastructure/production` workspace: `default`
 
 ### 1. dir: `infrastructure/staging` workspace: `default`
-<details><summary>Show Output</summary>
```

Comment:
wrapped results need `maxUnwrappedLines = 12`. After `TrimSpace`, this results become unwrapped.